src/python/tensorflow_cloud/core/containerize.py

```diff
@@ -157,32 +157,20 @@ def _create_docker_file(self):
                 if float(v[0] + "." + v[1]) <= 2.1:
                     self.docker_base_image += "-py3"
 
-        if not self._base_image_exist():
+        # TF nightly image names have the substring "dev"
```

Comment:
Do you mean [`tensorflow/tensorflow:latest-devel`](https://hub.docker.com/layers/tensorflow/tensorflow/latest-devel/images/sha256-3d7811876d1e6ffa85280d2238c0cd3cc1076f97649ca81984579954065eb95e?context=explore)?